.gitignore

```diff
@@ -43,3 +43,5 @@ testbin/
 # terraform
 .terraform
 
+# md
+README_when_create_plugin.md
```

Comment:
Please check your editor/IDE settings, to ensure EOF or a new empty line at the end of files.